### PR TITLE
CLN: Drop FramesSequenceMappable (replaced by slicerator)

### PR DESCRIPTION
**Pull request by nkeim**
Addresses #191

### CODE REVIEW

File: pims/base_frames.py [hunk 1]
```diff
@@ -11,7 +11,6 @@
 import functools
 import contextlib
```
Comment (caspervdw) [inline pims/base_frames.py]: This is also superfluous

**Comment by caspervdw** [comment]
Thanks. Couls you also drop the _index_generator? It appears that I missed it before.

**Comment by nkeim** [comment]
I removed all of the unused module imports (except `six`), and `_index_generator`.

**Comment by danielballan** [comment]
Thanks for the cleanup! `_index_generator` still gives me nightmares.

**Pull request merged**